srcpkgs/pipewire/template

```diff
@@ -1,14 +1,15 @@
 # Template file for 'pipewire'
 pkgname=pipewire
 version=0.3.20
-revision=1
+revision=2
 build_style=meson
 configure_args="-Dman=true -Dgstreamer=true -Ddocs=true -Dsystemd=false
  -Dbluez5=true -Dffmpeg=true -Dpipewire-alsa=true -Dpipewire-jack=true
  -Dudevrulesdir=/usr/lib/udev/rules.d"
-hostmakedepends="doxygen graphviz pkg-config xmltoman"
+hostmakedepends="doxygen graphviz pkg-config xmltoman gettext"
 makedepends="SDL2-devel ffmpeg-devel gst-plugins-base1-devel jack-devel
- sbc-devel v4l-utils-devel libva-devel libbluetooth-devel ncurses-devel"
+ sbc-devel v4l-utils-devel libva-devel libbluetooth-devel ncurses-devel
+ libopenaptx-devel fdk-aac-devel"
```

Comment:
Might make sense to leave a comment about the possibility of adding LDAC support when packaged?